### PR TITLE
add missing imports

### PR DESCRIPTION
**Pull request by shiranr**
Add analyzer as a dependency.
Fix imports

### CODE REVIEW

File: presidio-image-anonymizer/setup.py [hunk 1]
```diff
@@ -4,6 +4,11 @@
 
 from setuptools import setup, find_packages
 
+requirements = ["flask==1.1.2", "pillow", "pytesseract==0.3.7",
```
Comment (SharonHart) [inline presidio-image-anonymizer/setup.py:7]: flask is not a requirement for the package

File: presidio-image-anonymizer/Pipfile [hunk 2]
```diff
@@ -7,6 +7,7 @@ name = "pypi"
 flask = "==1.1.2"
```
Comment (biancafbmd) [inline presidio-image-anonymizer/Pipfile:7]: Should we also add pillow here too?

File: presidio-image-anonymizer/setup.py (hunk 1 above)
Comment (shiranr) [reply presidio-image-anonymizer/setup.py:7]: Removed.

File: presidio-image-anonymizer/Pipfile (hunk 2 above)
Comment (shiranr) [reply presidio-image-anonymizer/Pipfile:7]: Done.

File: presidio-image-anonymizer/setup.py [hunk 3]
```diff
@@ -4,29 +4,41 @@
 
 from setuptools import setup, find_packages
 
+requirements = ["pillow", "pytesseract==0.3.7", "presidio-analyzer>=1.9.0"]
```
Comment (SharonHart) [inline presidio-image-anonymizer/setup.py:7]: missing numpy?

File: presidio-image-anonymizer/setup.py [hunk 4]
```diff
@@ -4,29 +4,41 @@
...
+
+test_requirements = ["pytest>=3", "flake8==3.7.9", "flask==1.1.2"]
```
Comment (SharonHart) [inline presidio-image-anonymizer/setup.py:9]: drop flask here as well

File: presidio-image-anonymizer/setup.py [hunk 5]
```diff
@@ -4,29 +4,41 @@
...
 __version__ = ""
-parent_directory = os.path.abspath(
-    os.path.join(path.abspath(path.dirname(__file__)), os.pardir)
-)
+this_directory = path.abspath(path.dirname(__file__))
+parent_directory = os.path.abspath(os.path.join(this_directory, os.pardir))
+
+with open(path.join(this_directory, 'README.MD'), encoding='utf-8') as f:
+    long_description = f.read()
+
```
Comment (SharonHart) [inline presidio-image-anonymizer/setup.py:17]: you're not based

File: presidio-image-anonymizer/setup.py (hunk 4 above)
Comment (shiranr) [reply presidio-image-anonymizer/setup.py:9]: weird it's even here...

File: presidio-image-anonymizer/setup.py (hunk 5 above)
Comment (shiranr) [reply presidio-image-anonymizer/setup.py:17]: I was.

**Pull request closed without merging**